### ISSUE TITLE
Feature request: JSON support (especially for Shinmeikai dictionary)

### ISSUE THREAD

**Issue by honmashinsei** [rsimmons/voracious#34]
Love Voracious, it's great. Thank you for creating it.

I was trying to use the Shinmeikai J-J dictionary with it, which IMO is the best J-J dictionary out there. The definitions are short and awesome. However, it's not EPWING and/or is not supported by Yomichan's EPWING importer. However, I'm using it with Yomichan just fine, because it's in the JSON format.

Are you able to add JSON format support to Voracious? That would make my studies so much better! I'm using Daijirin in the EPWING format now, which has crappy definitions and is all over the place.

Thank you!

**Comment by SpongebobSquamirez** [comment]
新明解 epwings ARE floating around out there FYI (I have the EPWING version)

**Comment by honmashinsei** [comment]
> 新明解 epwings ARE floating around out there FYI (I have the EPWING version)

Yeah, I found it too, but try importing it. You'll get the error message
'The EPWING you selected is not supported (see instructions above)"

The instructions are here (Yomichan's):
https://foosoft.net/projects/yomichan-import/

There you'll see that 新明解 is not on that list. So it might be part of a bigger/Yomichan related issue.

**Comment by honmashinsei** [comment]
(However, you can use 新明解 with Yomichan by importing the JSON).
